rest/taskrouter/reservations/dequeue/example-1.cs

```diff
@@ -0,0 +1,27 @@
+// Download the twilio-csharp library from twilio.com/docs/csharp/install
+using System;
+using Twilio.TaskRouter;
+
+class Example
+{
+  static void Main(string[] args)
+  {
+    // Find your Account Sid and Auth Token at twilio.com/user/account
+    string AccountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+    string AuthToken = "your_auth_token";
+    string WorkspaceSid = "WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+    string TaskSid = "WTXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+    string ReservationSid = "WRXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+    var client = new TaskRouterClient(AccountSid, AuthToken);
+
+    // Update a Reservation with a Call instruction
+    Reservation reservation = client.GetReservation(WorkspaceSid, TaskSid, ReservationSid);
+    Console.WriteLine(reservation.ReservationStatus);
+    Console.WriteLine(reservation.WorkerName);
+
+    client.UpdateReservation(WorkspaceSid, "Tasks", TaskSid, ReservationSid,
+        instruction: "dequeue",
+        dequeueTo: "+12345678901",
```

Comment:
The other languages don't update this parameter and it is optional. Please remove it to maintain consistency across languages.